app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="sign_in_form" >
```

Comment:
[ OPTIONAL BUT RECOMMENDED ] User is unable to sign_in when clicking on the login button and entering valid credentials. Kindly make sure it works as expected.